Toggl.Foundation.MvvmCross/ViewModels/SelectClientViewModel.cs

```diff
@@ -27,16 +27,12 @@ public sealed class SelectClientViewModel : MvxViewModel<SelectClientParameters,
         private long workspaceId;
         private long selectedClientId;
         private SelectableClientViewModel noClient;
-        private ISubject<string> clientFilterText = new BehaviorSubject<string>(string.Empty);
 
         public IObservable<IEnumerable<SelectableClientBaseViewModel>> Clients { get; private set; }
-
+        public readonly ISubject<string> FilterText = new BehaviorSubject<string>(string.Empty);
```

Comment:
Don't use public fields, instead use a property
```suggestion
        public ISubject<string> FilterText { get; } = new BehaviorSubject<string>(string.Empty);
```